tests/unit/integrations/gcp/orchestrators/test_vertex_orchestrator.py

```diff
@@ -125,3 +129,96 @@ def test_vertex_orchestrator_stack_validation(
             artifact_store=gcp_artifact_store,
             container_registry=gcp_container_registry,
         ).validate()
+
+
+@pytest.mark.parametrize(
+    "resource_settings, orchestrator_resource_settings, expected_resources",
+    [
+        # ResourceSettings specified for the step, should take values from here
+        (
+            ResourceSettings(cpu_count=1, gpu_count=1, memory="1GB"),
+            {"cpu_limit": None, "gpu_limit": 1, "memory_limit": "1G"},
```

Comment:
Since GPU and memory have the same values for both settings, those might not necessarily come from the ResourceSettings

Reply:
updated!